testem-electron.js

```diff
@@ -1,14 +1,29 @@
 /* eslint-env node */
 module.exports = {
-  'test_page': 'tests/index.html?hidepassed',
-  'disable_watching': true,
-  'launchers': {
-    'Electron': require('./lib/test-support/test-runner'),
+  test_page: 'tests/index.html?hidepassed',
+  disable_watching: true,
+  launchers: {
+    Electron: require('./lib/test-support/test-runner'),
   },
-  'launch_in_ci': [
+  launch_in_ci: [
     'Electron',
   ],
-  'launch_in_dev': [
+  launch_in_dev: [
     'Electron',
   ],
+  browser_args: {
+    Electron: {
+      ci: [
+        // --no-sandbox may be needed when running inside a container
+        process.env.CI ? '--no-sandbox' : null,
+        '--headless',
```

Comment:
This doesn't seem to work. e.g. `ember electron --- --headless` still launches a visible window.